release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -2645,3 +2645,72 @@ Issued: 2020-06-15
 An update for 	openshift-enterprise-hyperkube-container is now available for
 {product-title} 4.4. Details of the update are documented in the
 link:https://access.redhat.com/errata/RHSA-2020:2449[RHSA-2020:2449] advisory.
+
+[[ocp-4-4-9-features]]
+==== Features
+
+[id="ocp-4-4-9-ibm-power"]
+===== IBM Power Systems
+
+With this release, IBM Power Systems are now compatible with {product-title} {product-version}. See xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on IBM Power] or xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on IBM Power in a restricted network].
+
+*Restrictions*
+
+Note the following restrictions for {product-title} on IBM Power:
+
+* {product-title} for IBM Power Systems does not include the following Technology Preview features:
+** Container-native virtualization (CNV)
+** {product-title} Serverless
+
+* The following {product-title} features are unsupported:
+** Red Hat OpenShift Service Mesh
+** OpenShift Do (odo)
+** CodeReady Containers (CRC)
+** {product-title} Pipelines based on Tekton
+** {product-title} Metering
+** SR-IOV CNI plug-in
+
+* Worker nodes must run Red Hat Enterprise Linux CoreOS.
+* Persistent storage must be of the `Filesystem` mode using local volumes, Network File System (NFS), OpenStack Cinder, or Container Storage Interface (CSI).
+* Networking must use either DHCP or static addressing with Red Hat Openshift SDN
+
+[id="ocp-4-4-9-ibm-z"]
+===== IBM Z and LinuxONE
+
+With this release, IBM Z and LinuxONE is now compatible with {product-title} {product-version}.
+See xref:../installing/installing_ibm_z/installing-ibm-z.adoc#installing-ibm-z_installing-ibm-z[Installing a cluster on IBM Z and LinuxONE] for installation instructions.
+
+*Restrictions*
+
+Note the following restrictions for {product-title} on IBM Z and LinuxONE:
+
+* {product-title} for IBM Z does not include the following Technology Preview features:
+** Container-native virtualization (CNV)
+** Log forwarding
+** Precision Time Protocol (PTP) hardware
+** CSI volume snapshots
+** CSI volume cloning
+** {product-title} Pipelines
+
+* The following {product-title} features are unsupported:
+** Red Hat OpenShift Service Mesh
+** OpenShift Do (odo)
+** CodeReady Containers (CRC)
+** {product-title} Metering
+** Multus CNI plug-in
+** {product-title} upgrades phased rollout
+** FIPS cryptography
+** Encrypting data stored in etcd
+** Automatic repair of damaged machines with machine health checking
+** Tang mode disk encryption during {product-title} deployment
+** {product-title} Serverless
+** Helm command-line interface (CLI) tool
+** Controlling overcommit and managing container density on nodes
+** etcd cluster operator
```

Comment:
More just a question as this confuses me, how is the etcd cluster operator not supported... it's the way that etcd members are scaled and controlled in a 4.4 cluster.

Reply:
I will ask.